src/libraries/System.Runtime/tests/System.IO.Tests/StreamReader/StreamReader.StringCtorTests.cs

```diff
@@ -40,6 +36,42 @@ public static void EmptyPath_ThrowsArgumentException()
             AssertExtensions.Throws<ArgumentException>("path", () => new StreamReader("", Encoding.UTF8, true, -1));
         }
 
+        [Fact]
+        public static void NullEncodingParamInCtor_ShouldNotThrowException()
+        {
+            Action ctorParamsStringEncoding = () => new StreamReader("", (Encoding)null);
+            var ctorParamsStringEncodingEx = Record.Exception(ctorParamsStringEncoding);
+            Assert.Null(ctorParamsStringEncodingEx);
+
+            Action ctorParamsStringEncodingBool = () => new StreamReader("", (Encoding)null, false);
+            var ctorParamsStringEncodingBoolEx = Record.Exception(ctorParamsStringEncodingBool);
+            Assert.Null(ctorParamsStringEncodingBoolEx);
+
+            Action ctorParamsStringEncodingBoolOptions = () => new StreamReader("", (Encoding)null, false, new FileStreamOptions());
+            var ctorParamsStringEncodingBoolOptionsEx = Record.Exception(ctorParamsStringEncodingBoolOptions);
+            Assert.Null(ctorParamsStringEncodingBoolOptionsEx);
+
+            Action ctorParamsStringEncodingBoolInt = () => new StreamReader("", (Encoding)null, false, 100);
+            var ctorParamsStringEncodingBoolIntEx = Record.Exception(ctorParamsStringEncodingBoolInt);
+            Assert.Null(ctorParamsStringEncodingBoolIntEx);
+
+            Action ctorParamStreamEncoding = () => new StreamReader(new MemoryStream(), null);
+            var ctorParamStreamEncodingEx = Record.Exception(ctorParamStreamEncoding);
+            Assert.Null(ctorParamStreamEncodingEx);
+
+            Action ctorParamStreamEncodingBool = () => new StreamReader(new MemoryStream(), null, false);
+            var ctorParamStreamEncodingBoolEx = Record.Exception(ctorParamStreamEncodingBool);
+            Assert.Null(ctorParamStreamEncodingBoolEx);
+
+            Action ctorParamStreamEncodingBoolInt = () => new StreamReader(new MemoryStream(), null, false, 100);
+            var ctorParamStreamEncodingBoolIntEx = Record.Exception(ctorParamStreamEncodingBoolInt);
+            Assert.Null(ctorParamStreamEncodingBoolIntEx);
+
+            Action ctorParamStreamEncodingBoolIntBool = () => new StreamReader(new MemoryStream(), null, false, 100, false);
```

Comment:
Can't you just call the ctor? Any exception will fail the test.

Reply:
Thanks! Done updating the test and call the constructor instead as suggested.
Please review, @danmoseley

Reply:
Tests look fine, I'll let someone in the area sign off though.

Reply:
@danmoseley 
thanks! 👍